register.js

```diff
@@ -17,9 +17,9 @@
     console.log("password =", orgPassword.value);
   });
 
-  if (orgPassword != cnfPassword) {
-    alert("Password and Confirm password should be same");
-  }
+  // if (orgPassword != cnfPassword) {
+  //   alert("Password and Confirm password should be same");
+  // }
```

Comment:
Why did you comment the code?

Reply:
This is for testing purpose and i will fix it ASAP.